tuning/data/data_config.py

```diff
@@ -148,28 +149,84 @@ def _validate_dataprocessor_config(dataprocessor_config) -> DataPreProcessorConf
         streaming = kwargs["streaming"]
         assert isinstance(streaming, bool), f"streaming: {streaming} should be a bool"
         c.streaming = streaming
-    if "chat_template" in kwargs:
+
+    provided_chat = "chat_template" in kwargs and kwargs["chat_template"] is not None
```

Comment:
Minor nit...can we rename these variables to

`is_chat_template_present`
`is_chat_template_path_present`
`is_chat_template_b64_present`